pkgs/development/interpreters/duktape/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "duktape-${meta.version}";
+  src = fetchurl {
+    url = "http://duktape.org/duktape-${meta.version}.tar.xz";
+    sha256 = "050csp065ll67dck94s0vdad5r5ck4jwsz1fn1y0fcvn88325xv2";
+  };
+
+  buildPhase = ''
+    make -f Makefile.sharedlibrary
+    make -f Makefile.cmdline
+  '';
+  installPhase = ''
+    install -d $out/bin
+    install -m755 duk $out/bin/
+    install -d $out/lib
+    install -m755 libduktape* $out/lib/
+  '';
+  enableParallelBuilding = true;
+
+  meta = {
+    version = "2.2.0";
+    description = "Duktape is an embeddable Javascript engine, with a focus on portability and compact footprint";
```

Comment:
No duktape in description. See https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.

Reply:
Right, sorry. i just copypasted that without thinking